week-1-ts-ecosystem/w01e03-object-oriented-structure/src/types.ts

```diff
@@ -0,0 +1,24 @@
+export type ProductType = 'KUP TERAZ' | 'AUKCJA' | 'ODDAM ZA DARMO';
+
+export interface IPrice {
+  value: number;
+  currency: 'PLN' | 'USD' | 'EUR'; // Our shop handle only these currencies
+}
+
+export interface IProduct {
```

Comment:
Ten interface jest niepotrzebny - powtarza zadanie klasy `Product` to ona powinna być powiązana z koszykiem